Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/NodeRelationAnchorPoint.php

```diff
@@ -24,28 +24,28 @@
 class NodeRelationAnchorPoint implements \JsonSerializable
 {
     private function __construct(
-        public readonly string $value
+        public readonly int $value
     ) {
     }
 
     public static function create(): self
     {
-        return new self(UuidFactory::create());
+        return new self(-1);
```

Comment:
I don't like the `-1` value here as `null` would be a better one. But I understand you don't want to handle `null` in all places.

But, does the `create` method still make sense? Couldn't we simply remove it, as the `NodeRelationAnchorPoint` is now managed by the database?

Reply:
Yeah, I guess we should just not offer it, I will take care.